exercises/takudzwa_makoni/ex01_basics/src/pch.cpp

```diff
@@ -0,0 +1,5 @@
+// pch.cpp: source file corresponding to pre-compiled header; necessary for compilation to succeed
```

Comment:
You shouldn't need this. Make sure you `git rm` the file.

Reply:
removed pch